### DISCUSSION TITLE
Need tool to measure the distance.

### DISCUSSION THREAD

**Discussion by pavel4win** [FakeFishGames/Barotrauma#12540] [Ideas and suggestions]
Please add to game a tool to measure the distance. It is necessary when setting up a variety of sensors and detectors.

**Comment by Hazmat616** [comment]
Id love a tape measure tool you place on the grid like a wire and walking will tell you the distance to that point while visually showing a tape with grid lines connecting you to it.

**Comment by Ek-Videogames** [comment]
In sub editor you can press Alt+Mouse1 and drag.

It would be nice to have an ingame equivalent too.
